src/metamath.c

```diff
@@ -1563,28 +1554,28 @@ void command(int argc, char *argv[]) {
                 while (1) {
                   p = instr(p + 1, str2, chr(((vstring)(g_fullArg[2]))[i]));
                   if (!p) break;
-                  /* Put spaces around special one-char tokens */
+                  // Put spaces around special one-char tokens
                   let(&str2, cat(left(str2, p - 1), " ",
                       mid(str2, p, 1),
                       " ", right(str2, p + 1), NULL));
-                  /*p++;*/
-                  /* Even though space is always a separator, it can be used
-                     to suppress all default tokens.  Go past 2nd space to prevent
-                     infinite loop in that case. */
+                  // p++;
+                  // Even though space is always a separator, it can be used
+                  // to suppress all default tokens.  Go past 2nd space to prevent
+                  // infinite loop in that case.
```

Comment:
```suggestion
                  // Even though space is always a separator, it can be used
                  // to suppress all default tokens.  Go past 2nd space to prevent
                  // infinite loop in that case.
                  // p++;
```